sandbox/registration-processor-mz.properties

```diff
@@ -492,20 +498,41 @@ mosip.regproc.camel.bridge.eventbus.kafka.poll.frequency=100
 #MosipVerticleAPIManager
 mosip.regproc.camel.bridge.eventbus.kafka.group.id=camel-brdige-mz
 
+mosip.regproc.camelbridge.intercept-hotlisted-key=HOTLISTED
+mosip.regproc.camelbridge.pause-settings=[{"hotlistedReason": "operator","pauseFor": 432000,"defaultResumeAction": "RESUME_PROCESSING_AND_REMOVE_HOTLISTED_TAG","fromAddress": ".*"},{"hotlistedReason": "center","pauseFor": 600,"defaultResumeAction": "STOP_PROCESSING","fromAddress": "eventbus://packet-validator-new-bus-out"}]
+mosip.regproc.camelbridge.endpoint-prefix=eventbus://
+
+
 #packet-uploader-stage
 mosip.regproc.packet.uploader.eventbus.kafka.commit.type=single
 mosip.regproc.packet.uploader.eventbus.kafka.max.poll.records=100
 mosip.regproc.packet.uploader.eventbus.kafka.poll.frequency=100
 mosip.regproc.packet.uploader.eventbus.kafka.group.id=packet-uploader-stage
 mosip.regproc.packet.uploader.message.expiry-time-limit=${mosip.regproc.common.stage.message.expiry-time-limit}
 
+packet.uploader.stage=registration-processor-packet-uploader-stage
```

Comment:
config.server.file.storage.uri in bootstrap.properties of packet uploader stage is using the above property, please take care of this also with tomorrow's PR